src/components/Composer.vue

```diff
@@ -1418,6 +1429,17 @@ export default {
 			return this.mainStore.getSmimeCertificate(certificateId)
 		},
 
+		smimeSignCheck(value) {
+			this.wantsSmimeSign = value
+			if (value) {
+				this.smimeSignAliases.push(this.selectedAlias.emailAddress)
```

Comment:
you can have multiple aliases for one email address right?

Reply:
I also thought about using the `aliasId ?? accountId`. But what if there is a collision. Both id ranges are not mutually exclusive.